recipes/ipycytoscape/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ipycytoscape" %}
+{% set version = "0.0.1" %}
```

Comment:
I think you need to have 0.1.0 since the pypi version is 0.1.0!

Reply:
Oh no, you're right. Hum! Will fix it, thanks Sylvain <3